common/src/api/internal/shared.rs

```diff
@@ -339,7 +430,29 @@ impl From<UplinkConfig> for PortConfigV1 {
     }
 }
 
-/// Deprecated, use PortConfigV1 instead. Cannot actually deprecate due to
+// Deprecated - still present to support importing from older bootstores
```

Comment:
Nit: if this is a doc-comment it will be immediately visible in rust-analyzer.
```suggestion
/// Deprecated - still present to support importing from older bootstores
```